.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -17,6 +17,7 @@ inputs:
         options:
             - x86_64-unknown-linux-gnu
             - aarch64-unknown-linux-gnu
+            - aarch64-unknown-linux-musl
```

Comment:
what about `x86_64-unknown-linux-musl`?

Reply:
Didn't try to since it was not what I needed,
I agree it should be supported as well.